app/views/services_and_information/_grouped_links.html.erb

```diff
@@ -1,7 +1,7 @@
 <% grouped_links.each_with_index do |group, group_index| -%>
-  <nav class="govuk-grid-row" aria-labelledby="<%= group.title.parameterize %>">
+  <div class="govuk-grid-row" aria-labelledby="<%= group.title.parameterize %>">
     <div class="govuk-grid-column-one-third">
-      <h1 id="<%= group.title.parameterize %>"><%= group.title %></h1>
+      <h2 class="govuk-heading-m" id="<%= group.title.parameterize %>"><%= group.title %></h1>
```

Comment:
Can we use the heading component here? 

```suggestion
      <%= render "govuk_publishing_components/components/heading", {
         font_size: "m",
         heading_level: 2,
         id: group.title.parameterize,
         text: group.title,
       } %>
```

As an added bonus we don't need to worry about the mismatch between the opening `h2` and the closing `h1`.

Reply:
I made the exact same mistake again 😂  Good spot, will fix.